integration_tests/.scripts/integration_tests.sh

```diff
@@ -23,13 +23,11 @@ fi
 
 for db in ${DATABASES[@]}; do
 
-  if  [ $db == 'bigquery' ]; then
-    echo "Snowplow web integration tests: Seeding data and doing first run"
-
-    eval "dbt seed --target $db --full-refresh" || exit 1;
-
-    eval "dbt run --target $db --full-refresh" || exit 1;
+  if [[ "$db" == "bigquery" || "$db" == "spark_iceberg" ]]; then
```

Comment:
Guessing the above is only due to local testing changes, let's discuss tomorrow if we are safe to leave int tests like this in general as if I understand correctly we only do more e2e tests for bigquery due to the more complicated structure (schema versioning) and for the rest we check the expected compiled sql based on normalized only (spark_iceberg not being part of that but I've noted below that the only diff is the catalog so I'd say it is fine for now.